.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Is there a reason to have .idea in the repo?

Reply:
by mistake; removed it in the last commit;